scan-command/src/main/java/io/ballerina/scan/internal/CoreRules.java

```diff
@@ -33,6 +34,7 @@ public class CoreRules {
 
     CoreRules() {
         coreRules = new ArrayList<>();
+        coreRules.add(RuleFactory.createRule(1, "Should avoid checkpanic", RuleKind.CODE_SMELL));
```

Comment:
Is this how descriptions usually are? Can't we just say

"Avoid 'checkpanic'"

Use ' anyway.

Reply:
There is an issue when printing the rules to the console when quotes are given, they resolve to a number format making the message as: `"Should avoid \u0027checkpanic\u0027"`